src/components/SQForm/SQFormTextarea.tsx

```diff
@@ -97,8 +100,8 @@ function SQFormTextarea({
         onChange={handleChange}
         onBlur={handleBlur}
         required={isFieldRequired}
-        minRows={rows}
-        maxRows={rowsMax}
+        minRows={minRows}
+        maxRows={maxRows}
         variant="outlined"
```

Comment:
Maybe not super-intuitive, but per the migration docs for the `TextField` component (https://mui.com/material-ui/migration/v5-component-changes/#textfield) the outlined variant is now the default variant so we can remove line 105.  Not absolutely necessary IMO just thought I would call it out.